boot-mongodb-elasticsearch/src/main/java/com/example/mongoes/web/service/RestaurantService.java

```diff
@@ -67,38 +67,33 @@ public Flux<Restaurant> loadData() throws IOException {
     }
 
     private Flux<Restaurant> saveAll(List<String> restaurantStringList) {
```

Comment:
@coderabbitai, can you please make the caller of this method `public Flux<Restaurant> loadData()`  as well reactive so that whole flow becomes reactive

Reply:
@coderabbitai, loadData method is throwing below error , please fix it

[1af4f87d-1] There was an unexpected error (type=Not Found, status=404).
No static resource .
org.springframework.web.reactive.resource.NoResourceFoundException: 404 NOT_FOUND "No static resource ."
	at org.springframework.web.reactive.resource.ResourceWebHandler.lambda$handle$1(ResourceWebHandler.java:432)
	Suppressed: The stacktrace has been enhanced by Reactor, refer to additional information below: 
Error has been observed at the following site(s):
	*__checkpoint ⇢ HTTP GET "/" [ExceptionHandlingWebHandler]

Reply:
Hi @coderabbitai,

I have fixed the issue with below code, but it is inserting only 8 instead of 32786 rows, please fix it


 public Flux<Restaurant> loadData() {
        return DataBufferUtils.read(
                        new ClassPathResource("restaurants.json"),
                        new DefaultDataBufferFactory(),
                        4096)
                .map(
                        dataBuffer -> {
                            byte[] bytes = new byte[dataBuffer.readableByteCount()];
                            dataBuffer.read(bytes);
                            DataBufferUtils.release(dataBuffer);
                            return new String(bytes, StandardCharsets.UTF_8);
                        })
                .flatMap(
                        fileContent -> {
                            List<String> restaurantArray = Arrays.asList(fileContent.split("\n"));
                            return this.saveAll(restaurantArray);
                        });
    }